src/pybind/mgr/diskprediction_local/requirements.txt

```diff
@@ -1,3 +1,3 @@
-numpy==1.15.1
-scipy==1.1.0
-scikit-learn==0.19.2
+numpy==1.23.0
+scipy==1.8.1
+scikit-learn==1.1.1
```

Comment:
these changes should go to a dedicated commit. also, i am not convinced that we need to bump up these dependencies. they are pin'ed to these versions because they are "supported" versions. unless the major distros we are supposed to support package, for instance python3-numpy v1.23.0, we should bump up the numpy's version.

Reply:
Hi @tchaikov 

I understand that bumping versions could be a larger endeavor, but I hoped to get some updates to these libraries because there have been a bunch of good changes. Some of them are security-related other performance. But let's leave that for another person to create a commit when they change in supported distros.

Best regards
Daniel